pilot/pkg/security/authz/policy/helper.go

```diff
@@ -194,7 +199,7 @@ func SimpleAuthzPolicy(name string, namespace string) *model.Config {
 	}
 }
 
-func Verify(got *envoy_rbac.RBAC, want map[string][]string) error {
+func Verify(got *envoy_rbac.RBAC, want map[string][]string, arePrincipalsImportant bool) error {
```

Comment:
the `arePrincipalsImportant` reads quite strange (by reading the name I have no idea what it is going to do), can you use more descriptive name?

Reply:
This mean if we care about the true principal values (which is important for trust domain migration). What would you name it?